dbsync.py

```diff
@@ -153,16 +71,25 @@ def _run_geodiff(cmd):
         raise DbSyncError("geodiff failed!\n" + str(cmd))
 
 
-def _geodiff_create_changeset(driver, conn_info, base, modified, changeset):
-    _run_geodiff([config.geodiff_exe, "diff", "--driver", driver, conn_info, base, modified, changeset])
+def _geodiff_create_changeset(driver, conn_info, base, modified, changeset, ignored_tables):
+    if ignored_tables:
+        _run_geodiff([config.geodiff_exe, "diff", "--driver", driver, conn_info, "--skip-tables", ignored_tables, base, modified, changeset])
```

Comment:
maybe pass this as arg to _run_geodiff or even would it be possible to read it from config and decide inside of _run_geodiff?